msg/DisplayRobotState.msg

```diff
@@ -3,3 +3,7 @@ RobotState state
 
 # Optionally, various links can be highlighted
 ObjectColor[] highlight_links
+
+# Optionally, hide the robot state if true.
+# This is useful in Rviz to programmatically toggler robot visualization
```

Comment:
```suggestion
# This is useful in Rviz to programmatically toggle robot visualization
```
Do we need this very specific comment at all?

Reply:
DisplayRobotState is really only used in Rviz, so i don't think its too specific. The more documentation, the better...

Reply:
No it's not. Actually, here you are refering to non-rviz C++ code that programmatically toggles visualization. Mentioning possible usages of a variable is too verbose and too specific. Future developers might come up with other uses of this variable ;-)